tests/test_conda_lock.py

```diff
@@ -2293,3 +2301,21 @@ def test_cli_version(capsys: "pytest.CaptureFixture[str]"):
     # the part before, in this case just "0.11.3".
     version_without_dev = __version__.split(".dev")[0]
     assert version_without_dev in result.stdout
+
+
+def test_pip_finds_recent_manylinux_wheels(
+    monkeypatch: "pytest.MonkeyPatch", lightgbm_environment: Path, conda_exe: str
+):
```

Comment:
Thanks so much @lesteve! Would the following docstring be accurate?

```suggestion
):
    """Ensure that we find a pre-built manylinux wheel for lightgbm.

    If not, installation would trigger a build of lightgbm from source. If this test
    fails, it likely means that MANYLINUX_TAGS in `conda_lock/pypi_solver.py` is out of
    date.
    """
```